sonar-project.properties

```diff
@@ -4,15 +4,18 @@ sonar.organization=govuk-one-login
 # Source files
 sonar.sources=src/
 sonar.exclusions=\
+  src/**/*.test.js,\
   src/assets/**,\
   src/app/**/fields.js,\
   src/app/**/steps.js,\
   src/app/**/index.js,\
   src/lib/config.js,\
+  src/lib/helpers.js,\
```

Comment:
I don't think this exists in this repo? and in any case we would want to test helpers